front/pages/api/auth/[...nextauth].js

```diff
@@ -15,7 +15,7 @@ export const authOptions = {
   ],
   secret: process.env.NEXTAUTH_SECRET,
   callbacks: {
-    session: async ({ session, token, user }) => {
+    session: async ({ session, token }) => {
```

Comment:
Is there a way to keep the parameter (super useful when touching this codebase) but marking it as expected to be unused?

Reply:
`({ session, token, user: _user })` would do the trick

Reply:
I'm in favor of that so that we don't forget the signature of these functions

Reply:
I tend to agree